src/main/res/values/strings.xml

```diff
@@ -271,13 +275,15 @@
         <item>@string/language_fr</item>
         <item>@string/language_es</item>
         <item>@string/language_bn</item>
+        <item>@string/language_hi</item
```

Comment:
This is a Compile error. PLEASE check your code by running it before submitting a PR